package.json

```diff
@@ -187,6 +188,7 @@
 		"tslint": "^5.20.1",
 		"typescript": "^4.5.2",
 		"utf-8-validate": "^5.0.7",
+		"uuid": "^9.0.0",
```

Comment:
There is a built-in library for this in NodeJS: https://nodejs.org/docs/latest-v16.x/api/crypto.html#cryptorandomuuidoptions.